docker-compose.yml

```diff
@@ -117,7 +117,7 @@ services:
     restart: always
     logging: *default-logging
   frontend:
-    image: lblod/frontend-lokaal-mandatenbeheer:0.1.1
+    image: lblod/frontend-lokaal-mandatenbeheer:0.1.2
```

Comment:
Where do you get this version from?
https://hub.docker.com/r/lblod/frontend-lokaal-mandatenbeheer/tags

* we should bump the version in the package.json to the latest tag aswell